packages/webdoc-default-template/tmpl/components/member/returns.tmpl

```diff
@@ -0,0 +1,20 @@
+<?js
```

Comment:
Again, description on top.

Reply:
Fixed in 9e7db62055baad1b7af9e1bcbdbfbd3fa02c5520.